src/core/meshcop/tcat_agent.hpp

```diff
@@ -351,7 +377,10 @@ class TcatAgent : public InstanceLocator, private NonCopyable
     ExtendedPanId                    mCommissionerExtendedPanId;
     char                             mCurrentServiceName[OT_TCAT_MAX_SERVICE_NAME_LENGTH + 1];
     State                            mState;
-    bool                             mAlreadyCommissioned : 1;
+    uint8_t                          mAdvertisement[OT_TCAT_ADVERTISEMENT_MAX_LEN];
+    char                             mVendorIdentifier[6];
```

Comment:
to use the defined constant instead of `6`

Reply:
fixed.